src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Performance/UseAsSpanInsteadOfRangeIndexer.Fixer.cs

```diff
@@ -94,6 +98,8 @@ private class UseAsSpanInsteadOfRangeIndexerCodeAction : CodeAction
                 _methodTarget = methodTarget;
                 _rangeArguments = rangeArguments;
                 EquivalenceKey = ruleId;
+                Title = string.Format(CultureInfo.InvariantCulture, MicrosoftNetCoreAnalyzersResources.UseAsSpanInsteadOfRangeIndexerCodeFixTitle, targetMethod,
```

Comment:
We should not use format on localized strings that passes non-localized strings as format arguments. Please define 2 separate resource strings here for string and array.